src/Dialogs/Task/Manager/TaskActionsPanel.hpp

```diff
@@ -55,6 +55,7 @@ class TaskActionsPanel : public RowFormWidget {
   void OnBrowseClicked();
   void OnNewTaskClicked();
   void OnDeclareClicked();
+  void OnDownloadClicked();
```

Comment:
New functions should be `noexcept` unless you know they can throw an exception.
Most of XCSoar's existing code doesn't do that, because we only allowed C++ exceptions in version 7.0. PRs for adding `noexcept` are very welcome, btw.